astropy/visualization/wcsaxes/core.py

```diff
@@ -724,6 +724,23 @@ def tick_params(self, axis='both', **kwargs):
                 if spine in coord.axislabels.get_visible_axes():
                     coord.tick_params(**kwargs)
 
+        def set_world_aspect(self, aspect, **kwargs):
+            """
+            Set the aspect of the axes in world coordinates.
+
+            Parameters
+            ----------
+            aspect : float
+                 A circle will be stretched such that the height is *aspect*
+                 times the width in world coordinates.
+            kwargs : dict
+                Keyword arguments are handed to
+                `matplotib.axes.Axes.set_aspect`.
+            """
+            cdelt = self.wcs.wcs.cdelt
```

Comment:
This is `astropy.wcs.WCS` specific, and there isn't really a concept of `cdelt` in the APE 14 API (http://docs.astropy.org/en/stable/api/astropy.wcs.wcsapi.BaseLowLevelWCS.html)

I am going to have to think about this a bit, and maybe @astrofrog will have a bright idea. I certainly think this is useful so we should work it out.

Reply:
Maybe one could do something like

```python
cdelt = pixel_to_world_values(2) - pixel_to_world_values(1)
```

This of course assumes that the pixel to world transformation is linear, so we should probably work out a way for checking for that somehow.

Reply:
Making it fully non-linear is going to involve nonuniformimage right?

Reply:
Oh yes, probably, but I was just thinking error out if it's not linear for now.